README.md

```diff
@@ -49,6 +64,7 @@ You can use environment variables in order to set up all options. You can set de
 - `BUGSNAG_HOSTNAME`
 - `BUGSNAG_APP_TYPE`
 - `BUGSNAG_APP_VERSION`
+- `BUGSNAG_IN_PROJECT_REGEX`
```

Comment:
I'm realizing now that allowing regex configuration from the env var has a downside: if you set a plain string in the config (like `config :bugsnag, in_project: "myproj"`), it's actually compiled as a regex, and regex characters will all have their special meanings. This seems surprising from a user standpoint.

I think a fine solution to this would be to say that if the config value is a string, it always uses `String.contains?` (and so this would just be `BUGSNAG_IN_PROJECT` or something). This removes a bit of flexibility from the default env var approach, but there's no reason users can't do something like this if they want to:

```elixir
config :bugsnag, in_project: fn({_, _, _, line}) -> line =~ Regex.compile(System.get_env("...")) end
```